### PR TITLE
Feat: Allow custom individual summary

### PR DESCRIPTION
**Pull request by KartikSoneji**
Allows for customized individual summaries read automatically from the `combined-summary.adoc` file in a catchup session.

### CODE REVIEW

**Comment by SirusCodes** [comment]
@KartikSoneji can you handle https://github.com/OurTechCommunity/catchup/blob/d6b9d63ad88ac6b887bccd734d3a7f5108b1ad75/summary/combined-summary.adoc#otc-catchup-96 this as well in this PR?😅

I completely forgot about this.

**Review by SirusCodes — APPROVED** [review]
I think it's ok after having handling 96 summary handling

**Comment by KartikSoneji** [comment]
Do we need this image still?

[`public/img/69th-catchup.png`](https://github.com/OurTechCommunity/catchup/blob/5464cf7e0e18560418b43af1553486b846c0d314/public/img/69th-catchup.png)

**Comment by SirusCodes** [comment]
> Do we need this image still?
> 
> [`public/img/69th-catchup.png`](https://github.com/OurTechCommunity/catchup/blob/5464cf7e0e18560418b43af1553486b846c0d314/public/img/69th-catchup.png)
> 
> 

Where it is used?👀

**Review by SirusCodes — APPROVED** [review]
LGTM

**Comment by KartikSoneji** [comment]
> > Do we need this image still?
> > [`public/img/69th-catchup.png`](https://github.com/OurTechCommunity/catchup/blob/5464cf7e0e18560418b43af1553486b846c0d314/public/img/69th-catchup.png)
> > 
> 
> Where it is used?👀

It was only added for the 69th catchup in 059102dedd7428225fd45dac0a61f9bfadae26e8

**Comment by SirusCodes** [comment]
It was never used anywhere in repo right?

Maybe it shouldn't be in that folder but let's keep it somewhere or use it. Because it's something which we created to celebrate 69th catchup. It's one of the memories which we should preserve 😅

**Comment by SirusCodes** [comment]
> > It was never used anywhere in repo right?
> 
> It was used for the redirect page in the week of the 69th CatchUp.
> 
> > Maybe it shouldn't be in that folder but let's keep it somewhere or use it. Because it's something which we created to celebrate 69th catchup. It's one of the memories which we should preserve 😅
> 
> Sure, we can add it to the 69th CatchUp's summary as a note.

@KartikSoneji can you open new issue for it

File: summary/sessions/096/combined-summary.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+😉 !eciN
+
+include::content.adoc[]
+
+include::attendees.adoc[]
+
+==== Meet Screenshot
+
+image::{catchup_number}.{catchup_image_extension}[alt="Meet #{catchup_display_number} screenshot", 600, 600]
```
Comment (SirusCodes) [inline summary/sessions/096/combined-summary.adoc:12]: Also `catchup_image_extension` is not yet supported as of this PR

File: summary/sessions/096/combined-summary.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[#{catchup_display_number}]
+=== link:/summary/{catchup_display_number}[OTC CatchUp #{catchup_display_number}]
```
Comment (SirusCodes) [inline summary/sessions/096/combined-summary.adoc:2]: We don't need `catchup_display_number` here right?

File: summary/sessions/096/combined-summary.adoc (hunk 1 above)
Comment (KartikSoneji) [reply summary/sessions/096/combined-summary.adoc:12]: Oh right.
Let's merge #86 ASAP then.

File: summary/sessions/096/combined-summary.adoc (hunk 2 above)
Comment (KartikSoneji) [reply summary/sessions/096/combined-summary.adoc:2]: We can hardcode it, but I think it's best to let the template remain the same.

Comment (SirusCodes) [reply summary/sessions/096/combined-summary.adoc:2]: Aren't we just wasting CPU cycles by not hard-coding it?

File: summary/sessions/096/combined-summary.adoc (hunk 1 above)
Comment (SirusCodes) [reply summary/sessions/096/combined-summary.adoc:12]: Yeah complete it as fast as you can

**Pull request merged**